TombEngine/Scripting/Internal/TEN/Effects/EffectsFunctions.cpp

```diff
@@ -303,7 +305,19 @@ namespace TEN::Scripting::Effects
 		TriggerBlood(pos.x, pos.y, pos.z, -1, USE_IF_HAVE(int, num, 1));
 	}
 
-/***Emit fire for one frame. Will not hurt Lara. Call this each frame if you want a continuous fire.
+/***Emit air bubble in a water room.
+@function EmitAirBubble
+@tparam Vec3 pos The position where the air bubble will be spawned.
+@tparam int roomNumber The room number where the air bubble will be generated.
+@tparam float size The size of the air bubble.
+@tparam float amplitude The oscillation amplitude of the air bubble.
+*/
+	static void EmitAirBubble(Vec3 pos, int roomNumber, float size, float amplitude)
```

Comment:
I recommend not to use `roomNumber` directly, e.g. `EmitParticle` implicitly uses `FindRoomNumber` to get room number from a given position. Also, 2 last arguments (`size` and `amplitude` should have default values and be optional.

Reply:
Done.